iceoryx2-ffi/c/Cargo.toml

```diff
@@ -18,6 +18,12 @@ build = "build.rs"
 path = "src/lib.rs"
 crate-type = ["rlib", "cdylib", "staticlib"] # without "rlib" the doc examples are not run
 
+[features]
+default = ["std"]
```

Comment:
Could you please also add the `std` feature flag to `iceoryx2/src/lib.rs` into the `Feature Flag` section of the documentation.